### ISSUE TITLE
Show traffic signals

### ISSUE THREAD

**Issue by AntonM030481** [organicmaps/organicmaps#2552]
If will be nice to see on map traffic signals for dots market as highway=traffic_signals.
Reference from Yandex.Maps.
![image](https://user-images.githubusercontent.com/85622715/168293836-0f2079f3-00d7-4b2b-9822-72be50776fdb.png)

BTW, this info can be useful for ETA.

**Comment by biodranik** [comment]
@vng они в данных есть?

**Comment by vng** [comment]
нет, давно стоит todo учитывать в весах traffic_signals, crossing, traffic_calming

**Comment by biodranik** [comment]
Так давай вставим в данные, можно же будет потом только в UI алгоритмы менять. Или нет?
Помню мы их отключили, чтобы не сильно замусоренная картинка была. Можно же включить, например, только в стиле для авто?

**Comment by vng** [comment]
Если говорить про ETA, weight - тут это не про добавить Feature. Добавить в данные - это если мы хотим посто видеть на карте.

**Comment by Jean-BaptisteC** [comment]
Can be closed, it's now available during navigation

**Comment by brdbkr** [comment]
I don't want to see traffic lights only during navigation as I never use navigation mode.

**Comment by RedAuburn** [comment]
> I never use navigation mode.
why not?

**Comment by brdbkr** [comment]
I also just tried out Organic Maps' navigation mode again on my drive home, and for me having it display track up with no option for north up is a _huge_ dealbreaker.

**Comment by patepelo** [comment]
Completed in #9313
